packages/flutter_driver/lib/src/extension/extension.dart

```diff
@@ -243,8 +243,6 @@ class FlutterDriverExtension {
 
   /// Runs `finder` repeatedly until it finds one or more [Element]s.
   Future<Finder> _waitForElement(Finder finder) async {
-    // TODO(mravn): This method depends on async execution. A refactoring
-    // for sync-async semantics is tracked in https://github.com/flutter/flutter/issues/16801.
     await Future<void>.value(null);
```

Comment:
My reading is that the TODO is tracking the removal of the (awkward) next line where we just await a future to make this method execute async. Is that no longer a problem since the issue is closed? Can we remove that line as well then?

See also https://github.com/flutter/flutter/pull/16798/files, where this TODO was added.

Reply:
I'm going to see if I can remvoe this line, if tests pass I'll land it - if that LGTY